app/assets/stylesheets/ltags/UserSubscriptionTag.scss

```diff
@@ -0,0 +1,85 @@
+.ltag__user-subscription-tag {
```

Comment:
You can skip right over this file if you'd like :trollface:. It's all hacky to get the MVP UX running for design to polish up 🎉 .

Reply:
_averts eyes knowing that we will refactor this 😄_